docs/modules/develop/pages/machine_translations.adoc

```diff
@@ -39,4 +39,11 @@ The class will need to be implemented, or reuse one from the community. Check th
 
 == Enabling the integration, organization-wise
 
-Each organization will be able to enable/disable machine translations if they want to. They can do that from the organization configuration.
+Each organization will be able to enable/disable machine translations if they want to. They can do that from the organization configuration. At the moment this isn't configured through the System dashboard (`/system`), so it needs to be configured through the Rails console:
```

Comment:
Each organization will be able to enable / disable the machine translations if they want to. The administrators of the organization perform the action from Settings -> Configuration admin menu, where they able to either enable or disable the machine translation system, but also they can select the priority.
* Original text first means that the platform will always display the original content as it has been added by contributors
* Translated text first means that the platform will always display the translation first.